mediacapture-streams/permission-helper.js

```diff
@@ -0,0 +1,10 @@
+async function setPermission(status="granted", scope=["camera", "microphone"]) {
+  try {
+    for (let s of scope) {
+      await test_driver.set_permission({ name: s }, status, true);
+    }
+  } catch {
```

Comment:
generic catch is usually a bad idea. What error do we expect here?

Reply:
this is to deal with browsers that don't implement the `set_permission` webdriver command (i.e. both Firefox and Safari at this time point).

Reply:
so we should catch "no such function". I'm worried about the case where it fails for an unrelated reason (such as calling with "scope=['cammera']" and getting a "no such value" exception - that could be hard to debug when the error is caught and ignored).